### DISCUSSION TITLE
HTTP API request fails with a Connection Reset

### DISCUSSION THREAD

**Discussion by jasper520** [rabbitmq/rabbitmq-server#4130] [Other]
When I deploy the service on the centos 7 server and log in to the console interface, I get a getFullYear error. It's not that my server lacks a running environment. The deployed version of rabbitmq-server is 3.9.13

```
main.js:1260 GET http://ip:15672/api/whoami net::ERR_CONNECTION_RESET
sync_req @ main.js:1260
sync_get @ main.js:1220
check_login @ main.js:106
(anonymous) @ main.js:63
nextRoute @ sammy-0.7.6.min.js:5
wrapped_route @ sammy-0.7.6.min.js:5
runRoute @ sammy-0.7.6.min.js:5
_checkFormSubmission @ sammy-0.7.6.min.js:5
(anonymous) @ sammy-0.7.6.min.js:5
listener_callback @ sammy-0.7.6.min.js:5
dispatch @ jquery-3.5.1.min.js:2
v.handle @ jquery-3.5.1.min.js:2
sammy-0.7.6.min.js:5 [Thu Feb 10 2022 19:51:59 GMT+0800 (中国标准时间)] body 500 Error put #/login Cannot read properties of undefined (reading 'getFullYear') TypeError: 500 Error put #/login Cannot read properties of undefined (reading 'getFullYear')
    at fmt_date0 (formatters.js:61:15)
    at fmt_date (formatters.js:47:15)
    at update_status (main.js:1152:13)
    at check_bad_response (main.js:1326:9)
    at sync_req (main.js:1272:9)
    at sync_get (main.js:1220:12)
    at check_login (main.js:106:23)
    at context_prototype.<anonymous> (main.js:63:13)
    at nextRoute (sammy-0.7.6.min.js:5:14189)
    at wrapped_route (sammy-0.7.6.min.js:5:14338)
```

**Comment by michaelklishin** [comment]
I will convert this issue to a GitHub discussion. Currently GitHub will [automatically close and lock the issue](https://github.com/github/feedback/discussions/3197) even though your question will be transferred and responded to elsewhere. This is to let you know that we do not intend to ignore this but this is how the current GitHub conversion mechanism makes it seem for the users :(

**Comment by michaelklishin** [comment]
"Connection reset" means a TCP connection was closed by the server. Start with [Troubleshooting Networking](https://rabbitmq.com/troubleshooting-networking.html).

[Clearing browser cache](https://rabbitmq.com/upgrade.html#rabbitmq-plugins-compatibility) is a recommended procedure between upgrades but I doubt it is relevant here.

**Comment by jasper520** [comment]
你好，来信收到，谢谢
